### PR TITLE
Fix calls from NET_RECEIVE.

### PR DESCRIPTION
**Pull request by 1uc**

### CODE REVIEW

File: src/codegen/codegen_neuron_cpp_visitor.cpp [hunk 1]
```diff
@@ -2129,6 +2129,10 @@ void CodegenNeuronCppVisitor::print_net_receive() {
     printer->add_line("auto * _ppvar = _nrn_mechanism_access_dparam(_pnt->prop);");
 
     printer->fmt_line("auto inst = make_instance_{}(_lmc);", info.mod_suffix);
+    printer->fmt_line("// nocmodl has a nullptr dereference for thread variables.");
+    printer->fmt_line("// NMODL will fail to compile at a later point, because of");
+    printer->fmt_line("// missing '_thread_vars'.");
```
Comment (JCGoran) [inline src/codegen/codegen_neuron_cpp_visitor.cpp:2134]: This can be a comment in NMODL, not the generated NMODL code.

Comment (1uc) [reply src/codegen/codegen_neuron_cpp_visitor.cpp:2134]: The point is to avoid a panic attack when reading the generated code.

Comment (pramodk) [reply src/codegen/codegen_neuron_cpp_visitor.cpp:2134]: No strong opinion on this. From users point of view,  until now we treated generated code as a "black box".  If we start adding more explanatory notes in the generated code, for potential debugging or understanding things, it's OK/better as well! (for devs as well)

**Review by pramodk — APPROVED** [review]
LGTM

File: src/codegen/codegen_neuron_cpp_visitor.cpp (hunk 1 above)
Comment (1uc) [reply src/codegen/codegen_neuron_cpp_visitor.cpp:2134]: It's only for ourselves. I can absolutely see myself panicking at my earlier stupidity if I see:
```
double * _thread = nullptr;
foo(..., _thread, ...);
```

**Pull request merged**